sdk/provisioning/Azure.Provisioning.AppService/tests/BasicAppServiceTests.cs

```diff
@@ -2,26 +2,21 @@
 // Licensed under the MIT License.
 
 using System.Threading.Tasks;
-using Azure.Core.TestFramework;
 using Azure.Provisioning.ApplicationInsights;
 using Azure.Provisioning.Expressions;
 using Azure.Provisioning.Resources;
 using Azure.Provisioning.Storage;
 using Azure.Provisioning.Tests;
-using Newtonsoft.Json.Linq;
 using NUnit.Framework;
 
 namespace Azure.Provisioning.AppService.Tests;
 
 public class BasicAppServiceTests(bool async)
-    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true /**/)
+    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true */)
```

Comment:
Not sure if this applies to others but for CI unit tests they should not inherit from ProvisioningTestBase.

Reply:
Fixed